packages/mendel-pipeline/package.json

```diff
@@ -22,10 +22,10 @@
     "cli-table": "^0.3.1",
     "commander": "^2.9.0",
     "debug": "^2.2.0",
-    "mendel-config": "^3.0.0-beta.2",
-    "mendel-deps": "3.0.0-beta.2",
```

Comment:
I think there is one package that used to be 2.0 before, I can't remember if this colides. Maybe mendel-core? you can `npm info` the ones that didn't change name and see if 2.0 will work or not.

Maybe we need all of them to be 3.0 because no package ever did become so high.